<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWindow</class>
 <widget class="QDialog" name="SettingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>310</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>310</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Default font color</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Default font size</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Start-up behaviour</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Special words source file</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_font_color">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>red</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>white</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>black</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>yellow</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>green</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>blue</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_font_size">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>24</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>32</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>72</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_start_behahvior">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Open new file</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Open most recent File</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Choose each time</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_specia_words_dir">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_select_sw_dir">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>250</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset settings</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>NotaPadQT Settings</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_highlight">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>391</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto highlight special words</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
